src/main/java/worker/Frontend.java

```diff
@@ -19,8 +20,12 @@
 
 
   ActorRef masterProxy = getContext().actorOf(
-      ClusterSingletonProxy.defaultProps("/user/master/active", "backend")
-      , "masterProxy");
+      ClusterSingletonProxy.props(
+          "/user/master",
+          ClusterSingletonProxySettings.create(getContext().system())
+              .withSingletonName("active")
```

Comment:
it could have been some paths that changed, but if you stick to defaults on both manager and proxy it should work